File: data/imported/knowledge/cells/cpp/merged/snippets/snippet_022.cpp
TEST(CellDataGoldenTests, xlsx_roundtrip_mixed_scalar_cells_file)
{
    TempDir temp("golden-file");
    const auto path = temp.Path("mixed.xlsx");
    auto workbook = CreateMixedCellWorkbook();
    workbook.Save(path.string());

    Workbook loaded(path.string());
    AssertMixedWorkbook(loaded, false);
}

File: data/imported/knowledge/cells/typescript/merged/snippets/snippet_022.ts
async function testWorksheetRename() {
  const workbook = new Workbook();

  workbook.worksheets[0]!.name = "RenamedSheet";
  console.log("Renamed to:", workbook.worksheets[0]!.name);

  await workbook.save("outputfiles/test_worksheet_rename.xlsx");
  console.log("Saved to outputfiles/test_worksheet_rename.xlsx");
}